Editor/UI/Drawers/MetadataWrapperDrawer.cs

```diff
@@ -228,10 +228,8 @@ private Rect DrawExtendableList(Rect rect, MetadataWrapper wrapper, Action<objec
                 return rect;
             }
 
-            Type elementType = (wrapper.Metadata[extendableListName] as ExtendableListAttribute.SerializedTypeWrapper).Type;
-
+            Type elementType = (wrapper.Metadata[extendableListName] as ExtendableListAttribute.SerializedTypeWrapper)?.Type;
```

Comment:
This just moves the NPE into DrawerLocator.GetInstantiatorDrawer which is actually it makes worse to find :P